### PR TITLE
Adds an enroll column to bulk enroll table

### PR DESCRIPTION
**Pull request by raq929**
I did a bit of moving components around because the SearchResults file was getting too unwieldy.

And I snuck in react-skeleton because I hate our loading screen.

Screen Shot 2021-03-10 at 5 26 42 PM

Screen Shot 2021-03-10 at 5 27 49 PM

### CODE REVIEW

File: src/components/BulkEnrollmentPage/CourseSearchResults.jsx [hunk 1]
```diff
@@ -122,33 +138,39 @@ export const BaseCourseSearchResults = ({
...
+        data={searchResults?.hits || []}
+        itemCount={searchResults?.nbHits}
+        isSelectable
+        pageCount={searchResults?.nbPages || 1}
+        pageSize={searchResults?.hitsPerPage || 0}
+        bulkActions={[{
+          buttonText: ENROLL_TEXT,
+          handleClick: handleBulkEnrollClick,
+        }]}
+        additionalColumns={[
```
Comment (binodpant) [inline src/components/BulkEnrollmentPage/CourseSearchResults.jsx:152]: didn't realize there was this idea of `additionalColumns`, how is it different from regular columns?

File: src/components/BulkEnrollmentPage/CourseSearchResultsCells.test.jsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+describe('CourseNameCell', () => {
+  const row = {
+    original: {
+      key: testCourseRunKey,
+    },
+  };
+  const slug = 'sluggy';
+  const wrapper = mount(<CourseNameCell value={testCourseName} row={row} enterpriseSlug={slug} />);
+  it('correctly formats a link', () => {
+    expect(wrapper.find('a').props().href).toEqual(`http://localhost:8734/${slug}/course/${row.original.key}`);
```
Comment (binodpant) [inline src/components/BulkEnrollmentPage/CourseSearchResultsCells.test.jsx:18]: forgot where this base url is setup, can this be a constant, or exported from somewhere?

**Review by binodpant — APPROVED** [review]
minor comments made: lgtm!

File: src/components/BulkEnrollmentPage/CourseSearchResults.jsx (hunk 1 above)
Comment (raq929) [reply src/components/BulkEnrollmentPage/CourseSearchResults.jsx:152]: It's a column that's not based on an accessor from the table data, basically. It has access to the row data so you can do what you want with it, but you can't really give it an accessor like you can with the other columns.

File: src/components/BulkEnrollmentPage/CourseSearchResultsCells.test.jsx (hunk 2 above)
Comment (raq929) [reply src/components/BulkEnrollmentPage/CourseSearchResultsCells.test.jsx:18]: yeah, it's probably a constant, I can change that.

**Pull request merged**